File: python/section2/q10.py
def load_pure_hightmp():
    DATADIR = '../../data/hightemp.txt'
    with open(DATADIR, 'r') as f:
        lines = f.readlines()
    return lines

if __name__ == '__main__':
    lines = load_pure_hightmp()
    print(len(lines))

File: scala/section2/src/main/scala/q10.scala
import scala.io.Source

/**
  * Created by nomotoeriko on 2017/06/08.
  */
object q10 {
  def main(args: Array[String]): Unit = {
    val sentence = load_sentence()
    println(sentence.length)
  }

  def load_sentence(path: String = "http://www.cl.ecei.tohoku.ac.jp/nlp100/data/hightemp.txt"): List[String] = {
    var sentence = List.empty[String]
    if ("http" == path.take(4)){
      for (line <- Source.fromURL(path).getLines()) sentence = sentence :+ line
    }else{
      for (line <- Source.fromFile(path).getLines()) sentence = sentence :+ line
    }
    sentence
  }
}
